src/boot/efi/assert.c

```diff
@@ -6,7 +6,11 @@
 #include "util.h"
 
 void efi_assert(const char *expr, const char *file, unsigned line, const char *function) {
-        log_error_stall(L"systemd-boot assertion '%a' failed at %a:%u, function %a(). Halting.", expr, file, line, function);
+        log_error("systemd-boot assertion '%s' failed at %s:%u, function %s(). Halting.",
```

Comment:
actually "systemd-boot" is a misleading here, might also be in "systemd-stub"...

Reply:
This is an assert message, and changing this is too much work for little gain. Nobody is expected to see this. The important thing is the mention of systemd so that the right people get blamed. :D